src/routes/docs/tutorials/nuxt/step-4/+page.markdoc

```diff
@@ -0,0 +1,173 @@
+---
+layout: tutorial
+title: Add authentication
+description: Add authentication to your Nuxt application using Appwrite Web SDK
+step: 4
+---
+
+# Nuxt composables {% #nuxt-composables %}
+
+Composables is a pattern in Nuxt to manage logic related to data fetching and global state management.
```

Comment:
We need just a little more context. I feel like this thought is incomplete, when do we use composables? Why are we creating one here?

Reply:
How is this related to authentication? Make sure to explain all of this so it makes sense to readers.

Reply:
Added some more context throughout the tutorial.